mixins/localize-dynamic-mixin.js

```diff
@@ -1,42 +1 @@
-import { getLocalizeOverrideResources } from '../helpers/getLocalizeResources.js';
-import { LocalizeMixin } from './localize-mixin.js';
-
-const fallbackLang = 'en';
-const supportedLangpacks = ['ar', 'cy', 'da', 'de', 'en', 'en-gb', 'es', 'es-es', 'fr', 'fr-fr', 'fr-on', 'hi', 'ja', 'ko', 'nl', 'pt', 'sv', 'tr', 'zh-cn', 'zh-tw'];
-
-export const LocalizeDynamicMixin = superclass => class extends LocalizeMixin(superclass) {
-
-	static async getLocalizeResources(langs, { importFunc, osloCollection, useBrowserLangs }) {
-
-		// in dev, don't request unsupported langpacks
-		if (!importFunc.toString().includes('switch') && !useBrowserLangs) {
-			langs = langs.filter(lang => supportedLangpacks.includes(lang));
-		}
-
-		for (const lang of [...langs, fallbackLang]) {
-
-			const resources = await importFunc(lang).catch(() => {});
-
-			if (resources) {
-
-				if (osloCollection) {
-					return await getLocalizeOverrideResources(
-						lang,
-						resources,
-						() => osloCollection
-					);
-				}
-
-				return {
-					language: lang,
-					resources
-				};
-			}
-		}
-	}
-
-	static get localizeConfig() {
-		return {};
-	}
-
-};
+export { LocalizeMixin as LocalizeDynamicMixin } from './localize/localize-mixin.js';
```

Comment:
Is `LocalizeDynamicMixin` now obsolete? It looks like `getLocalizeResources` has now been moved into `LocalizeMixin` itself? This is making my head hurt a bit... 😅

Reply:
Oops, looks like you answered this question in the description, my bad. I guess it's been a long day! 🙃 

My main concern is this is getting to be a lot of changes all wrapped up under the banner of "add localizeHTML method", it'll be pretty hard to see what actually changed...

Reply:
Normally I'd agree with that concern, but in this case I think it's all just very intertwined, and while we could do a bunch to make the history very clean I'm not sure it's worth it. The history will still be here via all these branches/PRs if we need to comb through it.

No objection if we do want to do that though.

Reply:
Yeah I agree, I'm not very comfortable with it but also at this point they are so closely related that doing them one at a time is going to add a lot of effort to re-sort through all the conflicts. When/if this does merge to `main` I do want to go through the other mixins and apply this same treatment.

Reply:
Haha okay... yolo!